app/ports/management/commands/load-rsync.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from ports.models import Port, LastPortIndexUpdate
+from parsing_scripts import rsync_operations
+
+
+class Command(BaseCommand):
+
+    help = "Populates the database with Initial data after fetching portindex.json from rsync"
+
+    def handle(self, *args, **options):
+        if not Port.objects.count() > 0:
+            # Fetch from rsync
+            rsync_operations.sync()
+
+            # Open the file
+            data = rsync_operations.open_file()
```

Comment:
What does "open file" have to do with "rsync operations"? The name rsync is super misleading: we could use rsync for just about anything.

You could have a class called `PortIndex` and then call `sync()` or `open()` on that one. And the `sync()` function could potentially call functionality from `RsyncHandler` class if necessary, but in that case `PortIndex` would specify what to sync from where, and the handler would only make sure of transferring the files.

Reply:
Yes, thank you. The naming here could be improved.